File: cpp/19a_1328A_DivisibilityProblem.cpp
#include<bits/stdc++.h>

int main(){
    int t, a, b, temp;
    std::vector<std::vector<int>> div;
    std::vector<int> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>a>>b;
        div.push_back({a,b});
        temp = b - (a % b);
        if(temp == b){
            temp = 0;
        }
        ans.push_back(temp);
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/19a_1328A_DivisibilityProblem.py
def main():
    t = int(input())
    ans = []
    for _ in range(t):
        a, b = input().split()
        a, b = int(a), int(b)
        temp = b - (a % b)
        if temp == b:
            temp = 0
        ans.append(temp)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()